.github/workflows/embedded-sdk-test.yml

```diff
@@ -0,0 +1,27 @@
+name: embedded-sdk-test-workflow
+
+on:
+  pull_request:
+    paths:
+      - "superset-embedded-sdk/**"
```

Comment:
In other similar flows we have slightly more specific `on`s to avoid CI running on Draft PRs. Something like this perhaps?
```
on:
  push:
    branches:
      - 'master'
  pull_request:
    paths:
      - "superset-embedded-sdk/**"
    types: [synchronize, opened, reopened, ready_for_review]
```

Reply:
nice, thanks!